src/components/Reports/AdditionalSalaryRequest/Shared/formCompletionPercentageHelper.ts

```diff
@@ -0,0 +1,19 @@
+import { CompleteFormValues } from '../CompleteForm/CompleteForm';
+
+export const formCompletionPercentageHelper = (
```

Comment:
"Helper" in a function name feels like it makes it longer without describing what the function does. What do you think about something like `calculateCompletionPercentage`? I usually prefer functions to start with verbs to emphasize what they do: e.g. "calculate".

```suggestion
export const calculateCompletionPercentage = (
```